src/package/packageVersion.ts

```diff
@@ -217,4 +231,78 @@ export class PackageVersion {
     updateResult.id = await pkgUtils.getSubscriberPackageVersionId(packageVersionId, this.connection);
     return updateResult;
   }
+
+  /**
+   * Increase the wait time for a package version that is org dependent.
+   *
+   * @param resolvedPackageId
+   * @param pollInterval
+   * @param wait
+   * @private
+   */
+  private async resolveOrgDependentPollingTime(
+    resolvedPackageId: string,
+    wait: Duration,
+    pollInterval: Duration
+  ): Promise<Duration> {
+    let w = wait;
+    // If we are polling check to seeif the package is Org-Dependent, if so, update the poll time
+    if (wait.milliseconds > 0) {
+      const query = `SELECT IsOrgDependent FROM Package2 WHERE Id = '${resolvedPackageId}'`;
+      try {
+        const pkgQueryResult = await this.connection.singleRecordQuery<PackagingSObjects.Package2>(query, {
+          tooling: true,
+        });
+        if (pkgQueryResult.IsOrgDependent) {
+          w = Duration.seconds((60 / pollInterval.seconds) * wait.seconds);
+        }
+      } catch {
+        // do nothing
+      }
+    }
+    return w;
```

Comment:
the let-free version
```suggestion
    // If we are polling check to see if the package is Org-Dependent, if so, update the poll time
    if (wait.milliseconds > 0) {
      const query = `SELECT IsOrgDependent FROM Package2 WHERE Id = '${resolvedPackageId}'`;
      try {
        const pkgQueryResult = await this.connection.singleRecordQuery<PackagingSObjects.Package2>(query, {
          tooling: true,
        });
        if (pkgQueryResult.IsOrgDependent) {
          return Duration.seconds((60 / pollInterval.seconds) * wait.seconds);
        }
      } catch {
        // do nothing
      }
    }
    return wait;
```